### PR TITLE
Add pdf of RNA-seq slides to repo and schedule

### PR DESCRIPTION
**Pull request by allyhawkins**
Here, I am adding the pdf of the slides for the intro to RNA-seq to the `slides` folder. I also added the link for these slides to the schedule. I chose to make a new line on the schedule for the introduction to RNA-seq and put that before the quantification notebook, but let me know if I should add them in to the schedule in another way.

### CODE REVIEW

File: workshop/SCHEDULE.md [hunk 1]
```diff
@@ -21,7 +21,8 @@ A tentative schedule for the September 2021 CCDL Virtual Training Workshop appea
 |             | [Consultation session](workshop-structure.md#consultation-sessions) | [Slack channel](../virtual-setup/slack-procedures.md#general-use) |
 | 5:00  PM    | End             |
 | **Day 2**   | **2021-09-21**  <br> [Introduction to bulk RNA-seq, Day 1: Raw data & exploratory analysis](https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/RNA-seq/README.md) | 
-| 12:00 PM    | Bulk RNA-seq: [QC, trim, and quantification with Salmon](https://htmlpreview.github.io/?https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/RNA-seq/01-qc_trim_quant.nb.html)| Zoom: [Main Session](../virtual-setup/zoom-procedures.md#joining-a-zoom-call) |
+| 12:00 PM    | Bulk RNA-seq: [Introduction (pdf)](../slides/2021-09-21_Intro_to_bulk_RNAseq.pdf)| Zoom: [Main Session](../virtual-setup/zoom-procedures.md#joining-a-zoom-call) |
+| 12:30 PM    | Bulk RNA-seq: [QC, trim, and quantification with Salmon](https://htmlpreview.github.io/?https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/RNA-seq/01-qc_trim_quant.nb.html)| Zoom: [Main Session](../virtual-setup/zoom-procedures.md#joining-a-zoom-call) |
```
Comment (jashapiro) [inline workshop/SCHEDULE.md:25]: I might not put a time here... I expect tehse two sections to flow together.
```suggestion
|             | Bulk RNA-seq: [QC, trim, and quantification with Salmon](https://htmlpreview.github.io/?https://github.com/AlexsLemonade/training-modules/blob/{{site.release_tag}}/RNA-seq/01-qc_trim_quant.nb.html)| Zoom: [Main Session](../virtual-setup/zoom-procedures.md#joining-a-zoom-call) |
```

**Review by jashapiro — COMMENTED** [review]
Looks good, but a couple little things to update: The skipped slides are still in the PDF, so you probably want to remove those, and the builds for slides 11-14 can probably be consolidated to a single slide (just 14).

I also suggested a tiny change to the schedule, but up to you how you want to handle that.

**Comment by allyhawkins** [comment]
Whoops, thanks for catching the skipped slides. I removed those and consolidated the animated slides 11-14 into one slide.

**Review by jashapiro — APPROVED** [review]
LGTM!

**Comment by jaclyn-taroni** [comment]
Closes #6

**Pull request merged**